CMakeLists.txt

```diff
@@ -47,8 +47,9 @@ option(NRN_ENABLE_CORENEURON "Enable CoreNEURON support" ${NRN_ENABLE_CORENEURON
 option(NRN_ENABLE_TESTS "Enable unit tests" ${NRN_ENABLE_TESTS_DEFAULT})
 # note that if CoreNEURON is enabled then it is not necessary to enable this option
 option(NRN_ENABLE_MOD_COMPATIBILITY "Enable CoreNEURON compatibility for MOD files" ${NRN_ENABLE_MOD_COMPATIBILITY_DEFAULT})
-option(NRN_USE_REL_RPATH "Use relative RPATH in binaries. for relocatable installs/Python" FALSE)
-mark_as_advanced(NRN_USE_REL_RPATH)
+option(NRN_ENABLE_REL_RPATH "Use relative RPATH in binaries. for relocatable installs/Python" ${NRN_ENABLE_REL_RPATH_DEFAULT})
+option(NRN_ENABLE_INTERNAL_READLINE "Use internal Readline library shipped with NEURON" ${NRN_ENABLE_INTERNAL_READLINE_DEFAULT})
+mark_as_advanced(NRN_ENABLE_REL_RPATH)
```

Comment:
Just curious. I know little about relocatable installs but it seems useful if robust. Eg. I have several times tried to
make a mac dmg that the user could use direct from the dmg or drag it anywhere to install. But always foundered for one reason or another. For example libnrniv.dylib not being relative to the launched program (eg. python).  It would
be cool for the pkg installer on mac to allow installation in a place other than /Applications and possible the post
install script could help with that. Also could be useful in future deb and rpm installers in case /usr/local is unsuitable
or not desired.

I agree that it is not necessary to mark NRN_ENABLE_INTERNAL_READLINE as advanced. But I suppose it could go either way.

Reply:
> I agree that it is not necessary to mark NRN_ENABLE_INTERNAL_READLINE as advanced. But I suppose it could go either way.
Just a note : this relative rpath was there, I just renamed it for a consistency.

Reply:
> Just curious. I know little about relocatable installs but it seems useful if robust. Eg. I have several times tried to make a mac dmg that the user could use direct from the dmg or drag the contents anywhere to install. 

Agree, that will be useful. But this relative rpath option has been added and tested in the context of wheel. I think we need a better testing (and may be some more fixes) to make it fully generic. I am also new to this installer stuff so not sure what all details involved.

> I agree that it is not necessary to mark NRN_ENABLE_INTERNAL_READLINE as advanced. But I suppose it could go either way.

I will keep this as it is then.

> I agree that it is not necessary to mark NRN_ENABLE_INTERNAL_READLINE as advanced. But I suppose it could go either way.

Just a note : this relative rpath was there, I just renamed it for a consistency.I